aws.yaml

```
@@ -210,7 +210,7 @@
     - name: etcd
       version: 3.3.15
 - version: 9.2.2
-  state: wip
+  state: active
   date: 2020-03-27T12:00:00Z
```

```suggestion
  date: 2020-04-01T12:00:00Z
```
Update to today :crossed_fingers: